druid/examples/styled_text.rs

```diff
@@ -131,4 +133,5 @@ fn ui_builder() -> impl Widget<AppData> {
         .with_child(mono_checkbox)
         .with_spacer(8.0)
         .with_child(input.padding(5.0))
+        .debug_widget_id()
```

Comment:
I'm assuming this is intentional to leave in? Flagging in case it's left over from development.